Task2/Locators.cs

```diff
@@ -0,0 +1,91 @@
+﻿namespace Task2
+{
+    class Locators
```

Comment:
it is not a good way of storing locators

it is better to use pattern PageObject. And store all needed elements and actions there